### ISSUE TITLE
The MTB module seems to be implemented wrong?

### ISSUE THREAD

**Issue by ZhengLeon** [Jaychoukobbe/GaitPart#4]
Hi, @Jaychoukobbe !
`            
x_part = x[:,:,:,i].permute(0, 2, 1).contiguous()
mtb1_attention = self.conv_list_1[i](x_part).sigmoid()
tmp_out_1 = self.max_pool1d_3(x_part) + self.avg_pool1d_3(x_part)
out_1 = tmp_out_1 * mtb1_attention
`
`x_part` is permuted to (batch, channel, frame), so the pool function is working on the  channel dimension data, which is different with the paper's 'sliding on sequence dimension'. I think the pool function should be applied to frame dimention data instead.

Thanks, hope for you reply~

**Comment by OliverHxh** [comment]
@ZhengLeon According to the official usage of [MaxPool1d](https://pytorch.org/docs/stable/generated/torch.nn.MaxPool1d.html), Maxpool1d function pools the input tensor along the 3-rd dimension, which matches the paper.

**Comment by ZhengLeon** [comment]
@Jaychoukobbe Thanks a lot, you are right!
